pkg/controller/component/rsm_convertor_test.go

```diff
@@ -20,9 +20,43 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 package component
 
 import (
+	appsv1alpha1 "github.com/apecloud/kubeblocks/apis/apps/v1alpha1"
+	workloadsalpha1 "github.com/apecloud/kubeblocks/apis/workloads/v1alpha1"
 	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
```

Comment:
make goimports

import (
<built-in pkgs>

<dot imorts>

<third party imports>

<same module imports>
)